reana_server/rest/users.py

```diff
@@ -427,7 +428,11 @@ def request_token():
         email_body = "New user access token request:\n\n" + "\n".join(
             [f"{f}: {getattr(user, f, None)}" for f in fields]
         )
-        send_email(ADMIN_EMAIL, email_subject, email_body)
+        try:
+            send_email(ADMIN_EMAIL, email_subject, email_body)
+        except REANAEmailNotificationError:
+            pass
```

Comment:
Instead of pass, could we log the error? Perhaps just printing it out to stdout/stderr, so that admins would see it when doing:

```
$ kubectl logs reana-server-5c5cdf658-ghv2z rest-api
```

Reply:
Also, currently, the error says:

```python
        raise REANAEmailNotificationError(
41-            "Cannot send email, missing server and port configuration. "
42-            "Please provide the following environment variables:\n"
43:            "REANA_EMAIL_SMTP_SERVER\nREANA_EMAIL_SMTP_PORT"
44-        )
```

If it could appear in User Web UI -- e.g. if we would merge only `r-ui` PR and not this one -- then we could reformulate it to something more user-friendly, like:

```
Error sending notification email. Missing REANA_EMAIL_SMTP_SERVER
and REANA_EMAIL_SMPT_PORT configuration. 
```

because users cannot "provide" it.

... that said, it would be quite a big change in the code base to both show the error and proceed with the user payload... we would probably need to change API response types... so if we only log this event on the server side, that might do, even without changing the error message.